response.go

```diff
@@ -28,19 +28,43 @@ import (
 )
 
 type Response struct {
-	RspObj    interface{}
-	Exception error
-	//Attachments map[string]string
+	RspObj      interface{}
+	Exception   error
+	Attachments map[string]string
+}
+
+// Pls just use NewResponse to get a Response.
+func NewResponse(rspObj interface{}, exp error, atta map[string]string) *Response {
```

Comment:
Use `exception` instead of `exp` and `attachment` instead of `atta`.

Reply:
ok